polymode.el

```diff
@@ -643,7 +643,9 @@ most frequently used slots are:
                                           (current-message)))))
                  (message ,(format "%s enabled" (concat root-name " polymode")))))
              (force-mode-line-update)
-             (pm--run-derived-mode-hooks ,config-name)
+             (let ((obj (clone ,config-name)))
+               (eieio-oset obj '-minor-mode ',mode)
```

Comment:
Thanks. There is certainly a bug, but the cloning should not be happening here. I don't recall why the logic in `pm--run-derived-mode-hooks` is so obfuscated but I think I have intended it to be run as part of `pm--run-init-hooks`. Will check.